app/assets/stylesheets/base/_variables.scss

```diff
@@ -62,8 +62,8 @@ $slate-900: #151921;
 $color-aliases: (
   "text-base": $charcoal-700,
   "text-lightest": $charcoal-400,
-  "text-link": $olive-700,
```

Comment:
Use the site and app's purple link color, so it matches the SVG arrow's color too.